### PR TITLE
Drop Node 6 support

### PR DESCRIPTION
**Pull request by Turbo87**
`ember-source@2.10.0-beta.1` has already dropped support for Node 6, which makes our test suite fail 😱 

Merging this PR will break our support policy by having `master` (and `beta`) not be compatible with Node.js 6 anymore before its official EOL (2019-04-30), but since our parent project has already broken that promise I guess this is fine...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ branches:
     - master
     - beta
     - release
+    - next
```
Comment (rwjblue) [inline .travis.yml]: Probably can remove?

Comment (rwjblue) [reply .travis.yml]: Though I guess it doesn't matter much...

Comment (Turbo87) [reply .travis.yml]: will remove it once this PR is merged. otherwise PRs against `next` will not run CI.

**Comment by Turbo87** [comment]
alright. we're doing this!

**Comment by rwjblue** [comment]
:tada:

**Comment by stefanpenner** [comment]
👯‍♂

**Pull request merged**